index.js

```diff
@@ -1,55 +1,76 @@
-const myStorage = window.localStorage;
-
-let books = (myStorage.getItem('books') !== null) ? JSON.parse(myStorage.getItem('books')) : [];
-
-books.forEach((data) => {
-  document.querySelector('#listbooks').innerHTML += `
-            <li>
-                <p>${data.title}</p>
-                <p>${data.author}</p>
-                <button class='removeBook' data-book-name="${data.title}" >remove</button>
-                <hr>
-            </li>
-        `;
-});
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-const addBook = (bookTitle, bookAuthor) => {
-  const book = {
-    title: '',
-    author: '',
-  };
-  if (bookTitle.value !== '' && bookAuthor.value !== '') {
-    book.title = bookTitle.value;
-    book.author = bookAuthor.value;
-    books.push(book);
-    document.querySelector('#listbooks').innerHTML += `
-            <li>
-                <p>${bookTitle.value}</p>
-                <p>${bookAuthor.value}</p>
-                <button type="button" data-book-name="${book.title}" onclick='removeBook(this)' >remove</button>
-                <hr>
-            </li>
-        `;
-    myStorage.setItem('books', JSON.stringify(books));
+  static db = () => ((window.localStorage.getItem('books') !== null) ? JSON.parse(window.localStorage.getItem('books')) : [])
+
+  static removeBook = (button) => {
+    const result = this.db().filter((word) => word.title !== button.getAttribute('data-book-name'));
+    button.parentElement.remove();
+    window.localStorage.setItem('books', JSON.stringify(result));
+    this.showAlert('Book deleted ', 'success');
```

Comment:
- [ ] The remove function should work just like before, unfortunately, I couldn't delete an item, I was getting an error "book is not undefined"